pyccel/ast/cupyext.py

```diff
@@ -142,7 +158,6 @@ class CupyArange(CudaNewArray):
         The type of the output array, if dtype is not given,
         infer the data type from the other input arguments.
     """
-    __slots__ = ('_start','_step','_stop','_dtype','_precision','_shape')
     _attribute_nodes = ('_start','_step','_stop')
     _rank = 1
     _order = None
```

Comment:
Having `_rank` and `_order` as class properties is good practice, however they cannot also be in `CupyNewArray`s slots

Reply:
Fixed in [07b3260](https://github.com/pyccel/pyccel/pull/1300/commits/07b3260ba94aa741a57d5dee713ad0fd1b7b3039).